7. a) La primera clase imprimirá:
valor final del try: 44
valor al final de finally: 45
Valor antes del return: 46
46.

b)imprimirá:
Valor final del catch: 43
valor al final de finally: 44
Valor antes del return: 45
45. Ya que no entra al try porque W no es un entero.

c)imprimirá:
valor al final de finally: 2
Excepcion en metodo () 
java.lang.NumberFormatException: For input string: "w"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at concurrente.ClassUno.metodo(ClassUno.java:18)
	at concurrente.ClassUno.main(ClassUno.java:31)

